Formula/file-roller.rb

```diff
@@ -51,6 +69,6 @@ def post_install
   end
 
   test do
-    system "#{bin}/file-roller", "--version"
+    assert_match version.to_s, shell_output("#{bin}/file-roller --version 2>/dev/null")
```

Comment:
That sounds optimistic about how careful people are with exit statuses. Poor hygiene on this could also mean poor hygiene on return value checking etc. Let's leave it visible. I'd only use the `>/dev/null` squashing if it's to silence output intentionally generated by the tested program itself, and even then, using a more specific `grep` filter might be better.